radio/apps/schedules/models.py

```diff
@@ -73,10 +74,9 @@ def clean(self):
 
             else:
                 # start_date != None and end_date == None only one can exist
-                qs = ScheduleBoard.objects.filter(
-                    start_date__isnull=False, end_date__isnull=True
-                ) | ScheduleBoard.objects.filter(
-                    end_date__gte=self.start_date
+                qs = (
+                    ScheduleBoard.objects.filter(start_date__isnull=False, end_date__isnull=True) |
+                    ScheduleBoard.objects.filter(end_date__gte=self.start_date)
```

Comment:
Make code more readable. The important part is to apply multiple filters, so the code should not spread this across multiple indentation levels, but should reflect this by intuitive reading.